dockerfiles/Dockerfile.router

```diff
@@ -0,0 +1,26 @@
+# Build is required to extract the release files
+FROM --platform=linux/amd64 alpine:latest AS build
+
+ARG DEBUG_IMAGE
+ARG ROUTER_RELEASE
+
+# Pull release from GH
+ADD https://github.com/apollographql/router/releases/download/v${ROUTER_RELEASE}/router-${ROUTER_RELEASE}-x86_64-linux.tar.gz /tmp/router.tar.gz
+
+WORKDIR /tmp
+
+# router.tar.gz extracts to "dist"
+RUN tar xvzf router.tar.gz -C /
+
+# Make directories for config and schema
+RUN mkdir /dist/config && mkdir /dist/schema
+
+# Final image uses distroless
+FROM --platform=linux/amd64 gcr.io/distroless/cc-debian11${DEBUG_IMAGE}
+
+# Copy in the extracted/created files
+COPY --from=build --chown=root:root /dist /dist
+
+WORKDIR /dist
+
+CMD ["./router", "--config", "./config/router.yaml", "--supergraph", "./schema/local.graphql"]
```

Comment:
Don't think we should specify the supergraph. Users will instead use configmap to set the env variables for this to be picked up from managed federation.

Reply:
done: [6b1fc58](https://github.com/apollographql/router/pull/764/commits/6b1fc58300a0bb98ec250d45d19f528f6ad58caf)